modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/DocumentLibrary/DocumentLibrary.es.js

```diff
@@ -48,17 +48,15 @@ function getDocumentLibrarySelectorURL({
 		'2_json': JSON.stringify(uploadCriterionJSON),
 		criteria:
 			'com.liferay.item.selector.criteria.file.criterion.FileItemSelectorCriterion',
-		doAsGroupId: themeDisplay.getScopeGroupId(),
 		itemSelectedEventName: `${portletNamespace}selectDocumentLibrary`,
 		p_p_auth: itemSelectorAuthToken,
 		p_p_id: Liferay.PortletKeys.ITEM_SELECTOR,
 		p_p_mode: 'view',
 		p_p_state: 'pop_up',
-		refererGroupId: themeDisplay.getScopeGroupId(),
 	};
 
 	const documentLibrarySelectorURL = createPortletURL(
-		themeDisplay.getLayoutRelativeControlPanelURL(),
+		`/group/guest/~/control_panel/manage?p_p_id=${Liferay.PortletKeys.DYNAMIC_DATA_MAPPING}`,
```

Comment:
@marcosapmf, what URL is returned by the method `themeDisplay.getLayoutRelativeControlPanelURL()`? Is the problem related to the URL or to the `groupId`?

I remember to talk to Paulino about this problem and we realized that the issue was associated to the `groupId`.

Reply:
@natocesarrego the problem is related to group but removing groupId from parameters isn't enough because `themeDisplay.getLayoutRelativeControlPanelURL()` returns `/group/forms/~/control_panel...` and `forms`path param in that url helps to decide with which group the document library will open.

Reply:
I see, @marcosapmf. Thanks for your clarification. 😉 

Can we have a more generic solution? Instead of have a hard coded URL, can we use the method `themeDisplay.getLayoutRelativeControlPanelURL()` and just replace the group info, e.g. replace `forms` by `guest`? This way if the default URL change, then we won't be affected.

Additionally, can you please check why on 7.1.x a [similar code](https://github.com/liferay/liferay-portal-ee/blob/f603a45f0355255b083e3c06745d1fd700d8eb90/modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/document-library/document_library_field.js#L49-L81) works and on master/7.2.x it doesn't work? I believe it'll help you find out the root cause of the issue.

Thanks.